### PR TITLE
integration page follow ups

### PR DESCRIPTION
**Pull request by KATT**
### Internals

- Replace `lodash.*` packages with plain `lodash` & replace `lodash.*` imports with `lodash/` - should have no impact on bundle size and opens up for us to use all of lodash
- Update `viewer.me` to cherry-pick what we actually need on that query to avoid leaking extra context info
- Update `getIntegrations` to never include `.key`-property to avoid leaking 

### Visual

- Update calendars so `primary` is displayed last
- Update connected calendars so they are in ascending order in which you connected them

## What needs testing

- `/event-types/[type]` as it uses `getIntegrations()`
- `/getting-started` as it uses `getIntegrations()`
- `/integrations` (again)

## Who needs to review this

- @zomars and/or @emrysal as they've recently touched the page that uses `getIntegrations()`
- @jimi-henning for testing

### CODE REVIEW

File: server/routers/viewer.tsx [hunk 1]
```diff
@@ -126,32 +160,34 @@ export const viewerRouter = createProtectedRouter()
...
+            const cals = await adapter.listCalendars();
+            const calendars = _(cals)
+              .map((cal) => ({
+                ...cal,
+                isSelected: !!user.selectedCalendars.find(
+                  (selected) => selected.externalId === cal.externalId
+                ),
+              }))
+              .sortBy(["primary"])
+              .value();
```
[the remark this answers is not in the document]
Comment (KATT) [reply server/routers/viewer.tsx:175]: From the homepage of https://lodash.com/



> ```js
> // Cherry-pick methods for smaller browserify/rollup/webpack bundles.
> var at = require('lodash/at');
> var curryN = require('lodash/fp/curryN');
> ```

[the remark this answers is not in the document]
Comment (KATT) [reply server/routers/viewer.tsx:175]: ... and on the server it doesn't really matter -- the above is server-side code

File: pages/api/integrations/googlecalendar/callback.ts [hunk 2]
```diff
@@ -25,8 +25,8 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     return;
   }
 
-  const { client_secret, client_id, redirect_uris } = JSON.parse(credentials).web;
-  const redirect_uri = process.env.GOOGLE_REDIRECT_URL || redirect_uris[0];
+  const { client_secret, client_id } = JSON.parse(credentials).web;
+  const redirect_uri = process.env.BASE_URL + "/api/integrations/googlecalendar/callback";
```
Comment (emrysal) [inline pages/api/integrations/googlecalendar/callback.ts:29]: 💯

**Review by emrysal — APPROVED** [review]
Tested through in the UI as well

**Pull request merged**